src/cloud/utils/rudderstack.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  RUDDERSTACK_DATA_PLANE_URL,
+  RUDDERSTACK_WRITE_KEY,
+} from 'src/shared/constants'
+import {ready, load} from 'rudder-sdk-js'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+
+function initializeRudderstack() {
+  ready(() => {})
+  load(RUDDERSTACK_WRITE_KEY, RUDDERSTACK_DATA_PLANE_URL)
+}
+
+if (isFlagEnabled('rudderstackReporting')) {
+  initializeRudderstack()
```

Comment:
If the goal of this is to only call it once, we should probably plop it onto a `useEffect` in a TSX file to initialize it when the top level component loads up and that's it, especially if we want this to load regardless of whether the user is logged in or not

Reply:
Is there a particular TSX file we are guaranteed to hit?  I picked this page as it's where logging in redirects you to by default as I presumed that was the normal user flow.

Reply:
the login page redirects you there if you haven't been given a link to be redirected to. I'd say top level of the app would be the TSX file that we're guaranteed to hit `App.tsx`

Reply:
Moved!

Reply:
Let’s sync in Monday